pyproject.toml

```diff
@@ -0,0 +1,21 @@
+[tool.poetry]
+name = "google-bigquery-data-modeller"
+version = "0.1.0"
+description = ""
+authors = ["Your Name <you@example.com>"]
+
+[tool.poetry.dependencies]
+python = "^3.7"
+jinja2 = "*"
```

Comment:
I have some new version freezes. Can we add them here?

Reply:
If you pull the latest master you'll see them.

Reply:
Yes, I'd be happy to do that!